publishing.md

```diff
@@ -53,8 +53,12 @@ Please make sure to update the destination of the source code link if you use a
 </span>
 ```
 
-## Changes
+## Distributing changes
 
-If you modify or extend OpenFisca, you are [legally required](https://choosealicense.com/licenses/agpl-3.0/) to make those changes available to the community. The easiest way to do it is to publish your fork, extension or reform on a source hosting platform such as GitHub, and to [notify](./community.md) the core team of this publication.
+If you modify or extend OpenFisca, and that you distribute those changes, you are [legally required](https://choosealicense.com/licenses/agpl-3.0/) to make those changes available to the community. The easiest way to do it is to publish your fork, country package, extension or reform on a source hosting platform such as GitHub, and to [notify](./community.md) the core team of this publication.
 
-Please note that, under AGPL provisions, serving over the network is considered as publishing changes, which means you also have to make your changes available even if you don't keep the running instance on your servers.
+Distributing changes means any way of sharing changes to the software with others. Providing software that integrate your changes in any format (source code, package, etc.) is considered as distribution, which means you also have to make your changes available.
+
+Please also note that, under AGPL provisions, serving over the network is considered as distribution, which means you also have to make your changes available, even if you don't keep the running instance on your servers.
```

Comment:
I don't understand “even if you don't keep the running instance on your servers”. If I'm serving (what?) over the network, am I not keeping the running instance on my servers? 🤔

Reply:
Hi @MattiSG, I don't understand either I haven't changed language of the phrase. However, look what a `git blame` threw as a result 😄:

    13b93a6f (Matti Schneider 2017-04-21 09:19:21 +0200 60) Please note that, under AGPL provisions, serving over the network is considered as publishing changes, which means you also have to make your changes available even if you don't keep the running instance on your servers.

Seems like you invented the phrase.

Reply:
What point are you trying to make exactly? 😕

Reply:
That I'm happy to remove the phrase 🙂.

Reply:
Thanks for that precision 😉 It's always hard to read intent through text, and it would certainly not be the first time that I re-read my own code and find it ridiculously bad. I don't find shame in correcting my old self (that certainly had good reasons for making mistakes, poor him!) and felt you were trying to inflict some. Happy to read I was wrong 🙂